calendar/calendar-impl/impl/src/java/org/sakaiproject/calendar/impl/GenericCalendarImporter.java

```diff
@@ -138,30 +139,19 @@ public class GenericCalendarImporter implements CalendarImporterService
 	
 	@Setter private FormattedText formattedText;
 
-	static DateFormat timeFormatter()
+	public static DateTimeFormatter timeFormatter()
 	{
-		DateFormat rv = new SimpleDateFormat("hh:mm a");
-		rv.setLenient(false);
-		return rv;
+		return DateTimeFormatter.ofPattern("h:mm[:ss] a");
 	}
 
-	static DateFormat timeFormatterWithSeconds()
+	public static DateTimeFormatter time24HourFormatter()
 	{
-		return new SimpleDateFormat("hh:mm:ss a");
+		return DateTimeFormatter.ofPattern("HH:mm[:ss]");
 	}
 
-	static DateFormat time24HourFormatter()
+	public static DateTimeFormatter dateFormatter()
 	{
-		DateFormat rv = new SimpleDateFormat("HH:mm");
-		rv.setLenient(false);
-		return rv;
-	}
-
-	static DateFormat time24HourFormatterWithSeconds()
-	{
-		DateFormat rv = new SimpleDateFormat("HH:mm:ss");
-		rv.setLenient(false);
-		return rv;
+		return DateTimeFormatter.ofPattern("M/d/yyyy");
```

Comment:
ewwwwww! who is using this format!

Reply:
yeah this was the format they were using I didn't change that

Reply:
I switched to using a localized ISO format, with a backup of m/d/y just in case!